tests/optimize/test_hyperopt.py

```diff
@@ -1160,7 +1160,7 @@ def test_simplified_interface_sell(mocker, default_conf, caplog, capsys) -> None
     out, err = capsys.readouterr()
     assert 'Best result:\n\n*    1/1: foo result Objective: 1.00000\n' in out
     assert dumper.called
-    # Should be called twice, once for tickerdata, once to save evaluations
+    # Should be called twice, once for historical candle data, once to save evaluations
```

Comment:
you must hate the term tickerdata very much to adjust all comments in tests!! :laughing:

Reply:
No, the only way to see that the "ticker" term is only left in the places when it really corresponds to the ticker is to substitute it with proper terms in all wrong places, including names of the local vars and comments. Otherwise the grep results will non be clear. Including the tests. Plus, since this usage is confusing -- the tests should also be clean and understandable, with proper terms used, including comments... Especially for future devs.